test/xpu/windows_skip_cases.py

```diff
@@ -5,8 +5,8 @@
 
 skip_dict = {
     # Windows: Skip entire files using *.py:: pattern
-    "test_decomp": [
-        "test_decomp.py::",  # Skip entire file on Windows
+    "test_decomp.py": [
```

Comment:
to align with the naming style you could change test_decomp.py to test_decomp_xpu.py.